components/authz-service/server/v2/project_update_manager.go

```diff
@@ -114,18 +124,28 @@ func NewWorkflowExecutor() (*patterns.ChainWorkflowExecutor, error) {
 	// we can generate the workflow executor for the correct domain service easily.
 	// The logic only depends on the name. This means that an instance of authz that
 	// didn't know about a domain service can still drive its workflow.
-	domainSvcExecutor := patterns.NewParallelWorkflowExecutor(func(key string) (cereal.WorkflowExecutor, bool) {
+	parallelWorkflowExecutor := patterns.NewParallelWorkflowExecutor(func(key string) (cereal.WorkflowExecutor, bool) {
 		lock.Lock()
 		defer lock.Unlock()
 		if workflowExecutor, ok := workflowMap[key]; ok {
 			return workflowExecutor, true
 		}
+
+		var workflowExecutor cereal.WorkflowExecutor
+		if key == serialSubworkflowName {
```

Comment:
This workflow is a parallel workflow. When we enqueue it, it will have subworkflows. Before, these were just the names of the services ("ingest", "nodemanager", "compliance"). Now, one of the subworkflows is the new workflow that does all the ES stuff in a serialized way, which is called `serialSubworkflowName`. This all just makes sure we use the correct logic for each subworkflow.